src/core/jwk.rs

```diff
@@ -263,7 +263,8 @@ impl CoreRsaPrivateSigningKey {
             return Err(format!("RSA private key must end with {}", RSA_FOOTER));
         }
         let base64_pem = &trimmed_pem[RSA_HEADER.len()..trimmed_pem.len() - RSA_FOOTER.len()];
-        let der = base64::decode_config(base64_pem, base64::MIME)
+        let base64_pem: String = base64_pem.chars().filter(|b| ![' ', '\n', '\t', '\r', '\x0b', '\x0c'].contains(b)).collect();
```

Comment:
it looks like https://tools.ietf.org/html/rfc7468 is the most relevant standard. [Figure 2 (Lax)](https://tools.ietf.org/html/rfc7468#section-3) is probably what we want here, and I think that's consistent with this code. mind adding a comment that this is RFC 7468 "Lax" parsing?

I'm curious about the relative performance of `[...].contains()` vs. a pre-compiled regex vs. Rust pattern matching, but it probably doesn't matter too much since this code doesn't seem like it would be on the critical path for most client applications.

Reply:
I added the comment and switched to pattern matching. Guess from those three pattern matching is the fastest.